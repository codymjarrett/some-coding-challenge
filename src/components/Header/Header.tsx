import React from 'react'

import { Link } from 'react-router-dom'

const Header: React.FC = () => {
	return (
		<header className="header">
			<div className="header__wrapper">
				<h1 className="header__title">
					<Link className="header__link" to="/">
						San Fran Movie Fun
					</Link>
				</h1>
			</div>
		</header>
	)
}

export default Header
