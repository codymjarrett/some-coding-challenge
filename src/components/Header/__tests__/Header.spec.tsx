import React from 'react'
import { render, cleanup } from '@testing-library/react'
import { BrowserRouter as Router } from 'react-router-dom'

import Header from '../Header'

afterEach(cleanup)

it('<Header />', () => {
	const { container, debug } = render(
		<Router>
			<Header />
		</Router>
	)
	expect(container.firstChild).toMatchSnapshot()
})
