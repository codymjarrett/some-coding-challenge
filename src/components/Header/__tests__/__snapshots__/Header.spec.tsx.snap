// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> 1`] = `
<header
  class="header"
>
  <div
    class="header__wrapper"
  >
    <h1
      class="header__title"
    >
      <a
        class="header__link"
        href="/"
      >
        San Fran Movie Fun
      </a>
    </h1>
  </div>
</header>
`;
